### DISCUSSION TITLE
aMule, Quazaa and Shareaza

### DISCUSSION THREAD

**Discussion by GreenLunar** [ansani/Shareaza#133] [General]
The only single software I miss ever since I migrated to Linux is Shareaza

There is a similar project called [aMule](https://sourceforge.net/projects/amule/) (of eMule) but it is not enough, and also [Quazza](https://sourceforge.net/projects/quazaa/) which appears inactive.

I suggest to join people from aMule, eMule and Quazza, and make a software that can run on any platform, and perhaps also on mobile devices.

**Comment by ansani** [comment]
Hi! All my releases of Shareaza can run on Linux / macOS using Wine. It's not possible to create a native release of Shareaza as the client uses the MFC framework. I started making a fork with UAP3, but it's in the pre-alpha stage.

**Reply by GreenLunar** [reply]
My kids use WineHQ, but I don't. I'm too traumatized of Windows ;)

**Comment by GreenLunar** [comment]
Please join us:
@amule-project @gonosztopi @sc0w
@quazaa-development-team @c3c @josephcrowell
